compiler/rustc_typeck/src/expr_use_visitor.rs

```diff
@@ -570,6 +571,38 @@ impl<'a, 'tcx> ExprUseVisitor<'a, 'tcx> {
         }));
     }
 
+    /// Walk closure captures but using `closure_caputes` instead
+    /// of `closure_min_captures`.
+    ///
+    /// This is needed because clippy uses `ExprUseVisitor` after TypeckResults
+    /// are written back. We don't currently writeback min_captures to
+    /// TypeckResults.
```

Comment:
Can you add a FIXME(project-rfc-2229#18) comment that this should be removed eventually

Reply:
https://github.com/rust-lang/rust/pull/78801/files/c50e57f946ee5a73b50fa5c52bb7a2a8a0cecf3f..bb8c5e5d8b4961a26f88b320f719249a9db8225e#diff-7e389c2ca4db213dcaee698d12a364e7e39f77179a0da1c6578e1ffc9187a9e2R581-R582